xml/ftp_banners.xml

```diff
@@ -1317,4 +1317,12 @@ more text</example>
     <param pos="2" name="service.version"/>
     <param pos="1" name="host.name"/>
   </fingerprint>
+  <fingerprint pattern="^Amazon Linux AMI release (.*)">
```

Comment:
use `\s` to match white space?

Reply:
Instead of match `(.*)`, I would match:
`^Amazon\sLinux\sAMI\srelease\s(\d+\.\d+)%`

Reply:
@cmccrisken-r7 I assume you mean `^Amazon\sLinux\sAMI\srelease\s(\d+\.\d+)$`? With `$` rather than `%`?

Reply:
Since the intention here is to match single whitespace characters then just a space in the pattern is more readable with no impact on matching. It also is more consistent with the rest of the patterns contained within the XML. But i'm open to either way

Reply:
:+1: